content/en/tools/cloud-pods/pods-cli/index.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Cloud Pods CLI"
+weight: 5
+categories: ["LocalStack Pro", "Tools", "Persistence", "CLI"]
+description: >
+  LocalStack provides a command line tool to manage the state of your instance via Cloud Pods.
+---
+
+## Syntax
+Use the following syntax to run `localstack pod` commands from your terminal window:
+
+```bash
+localstack pod [command] [options]
+```
+
+where `command` specifies the operation you want to perform with your Cloud Pods, e.g., `pull` or `push`, and `options` specifies the optional flag. 
+For example, you can attach a specific message to a snapshot using the `-m` option while doing a `commit` operation.
+
+## Configuration
+The CRUD commands exposed with the Cloud Pods CLI expect a `--name <pod_name>` option to specify the pod's name.
+Users can avoid specifying a pod name at every command by setting a global config with the `config` command.
+
+For instance, the following command
+```
+localstack config --name my_pod
+```
+will implicitly pass a pod name to all subsequent CLI commands. 
+Such a configuration will be saved locally on the host machine in a JSON file (e.g., in `~/.localstack/cloudpods/pods-config.json`).
+
+## Commands
+
```

Comment:
@whummer  had this wonderful figure showing what components the cli commands interact with. Could we add this here?

Reply:
Done, added it. In the section above, though, seems to fit better in the general description of cloud pods. Post v1 we can still move it down here (at least the right part of the figure that illustrates the pods operations). 👍